eng/Version.Details.xml

```diff
@@ -196,33 +196,33 @@
     </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="7.0.0-beta.22327.2">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="7.0.0-beta.22355.4">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>a264eb13fea14125f3ef8d4056586cd66fa55309</Sha>
+      <Sha>9f45238c23d89cf44a10705db1217e66a441ba5a</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.CMake.Sdk" Version="7.0.0-beta.22327.2">
+    <Dependency Name="Microsoft.DotNet.CMake.Sdk" Version="7.0.0-beta.22355.4">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>a264eb13fea14125f3ef8d4056586cd66fa55309</Sha>
+      <Sha>9f45238c23d89cf44a10705db1217e66a441ba5a</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
```

Comment:
```suggestion
```

@MichaelSimons does this look right?  We don't want duplicate `<SourceBuild>` nodes in the Version.Details.xml right?

Reply:
Good eye Chris, yes that is not correct and could be the cause of the error.